### PR TITLE
Docs/minor text edits

### PR DESCRIPTION
**Pull request by signekb**
## Description 
As I go through the course, I'll add minor text modifications to this branch, like fixing typos, layout, or adding missing words. These changes only include up to and including the functionals session. 

One thing that I noticed while going through the functionals session was that  *some* code chunks were a bit "blurry" while others were *not* (in my browser at least), see screenshot below. My best guess is the difference in how the code chunk was set up, so I aligned that in `functionals.Rmd`. However, I'm not really sure if this fixes the issue :) 

Screenshot 2024-05-04 at 17 43 05

### CODE REVIEW

File: sessions/functionals.qmd [hunk 1]
```diff
@@ -235,23 +237,23 @@ usethis::use_package("purrr")
...
 expression](https://r4ds.had.co.nz/strings.html#matching-patterns-with-regular-expressions)
 or also `regex`) to find the pattern, and `recurse` to look in all
-subfolders. We'll cover regular expressions more in the next session.
+subfolders. We'll cover regular expressions more in the next session. In
+our case, the pattern is `user_info.csv`, so the code should look like
+this:
 
-```{r list-user-info-files}
-#| filename: "doc/learning.qmd"
+``` {.r filename="doc/learning.qmd"}
```
Comment (lwjohnst86) [inline sessions/functionals.qmd:256]: When the `r` in the code chunk has a `.` before it, it tells Quarto to not run it. We actually want to run this code when building the website. So I am reverting this change.

File: sessions/functionals.qmd [hunk 2]
```diff
@@ -267,31 +269,29 @@ user_info_files
 ```
 
 ```{r admin-list-files-for-book}
-#| echo: false
 head(gsub(".*\\/data-raw", "data-raw", user_info_files), 3)
+print("...")
```
Comment (lwjohnst86) [inline sessions/functionals.qmd:273]: Hmm, I'm not sure I understand the reasoning for this change, but we don't want to include the code itself, but do want to include the output (we rarely need to use `print()` in R, unlike Python). I will revert this change ☺️

File: sessions/functionals.qmd [hunk 3]
```diff
@@ -267,31 +269,29 @@ user_info_files
...
 ```
 
 Alright, we now have all the files ready to give to `map()`. So let's
 try it!
 
-```{r}
-#| filename: "doc/learning.qmd"
+``` {.r filename="doc/learning.qmd"}
```
Comment (lwjohnst86) [inline sessions/functionals.qmd:279]: Same comment above, we want to run this code.

**Comment by lwjohnst86** [comment]
Hmm, that is really weird. I have never seen that before... and I don't understand why that happens. Neither myself nor Anders (on a few browsers) see this issue. Maybe we can check your computer tomorrow some time?

**Comment by lwjohnst86** [comment]
Mange tak!!!! 🤩

File: sessions/functionals.qmd (hunk 2 above)
Comment (signekb) [reply sessions/functionals.qmd]: Woops, that was also not something I wanted to commit - I was testing something! Great!

File: sessions/functionals.qmd (hunk 1 above)
Comment (signekb) [reply sessions/functionals.qmd]: Ah, makes sense!

**Pull request merged**